### ISSUE TITLE
Error: AttributeError: 'Events' object has no attribute 'request_success'

### ISSUE THREAD

**Issue by AbhiShrek** [sabyadi/mongolocust#9]
I followed the steps given in README.md
```
git clone https://github.com/sabyadi/mongolocust
cd mongolocust
python3 -m venv venv
source venv/bin/activate
pip install -r requirements.txt
```

Then added the URL of my cluster in `settings.py` file.
made the `BULK_INSERT_WEIGHT` and `AGG_PIPE_WEIGHT` to 0.

These are the only changes I made, and when I ran locust with 
`locust -f load_test.py --headless -u 2 -r 0.1 --run-time 10m`

its giving this error continuously and not generating load.
```
[2023-03-05 11:29:57,232] my-vm2-eus2/ERROR/locust.user.task: 'Events' object has no attribute 'request_success'
Traceback (most recent call last):
  File "/home/azureuser/work/temo/mongolocust/venv/lib/python3.8/site-packages/locust/user/task.py", line 347, in run
    self.execute_next_task()
  File "/home/azureuser/work/temo/mongolocust/venv/lib/python3.8/site-packages/locust/user/task.py", line 372, in execute_next_task
    self.execute_task(self._task_queue.pop(0))
  File "/home/azureuser/work/temo/mongolocust/venv/lib/python3.8/site-packages/locust/user/task.py", line 493, in execute_task
    task(self.user)
  File "/home/azureuser/work/temo/mongolocust/mongo_user.py", line 43, in run_mongodb_operation
    self.environment.events.request_success.fire(
AttributeError: 'Events' object has no attribute 'request_success'
```

What could be wrong ?

**Comment by AbhiShrek** [comment]
Found out whats wrong:
https://docs.locust.io/en/stable/changelog.html?highlight=request_success#id1
`request_success` is removed from the latest version.

**Comment by sabyadi** [comment]
merged PR #10 which resolves the issue.
